lib/sqitchtutorial-mysql.pod

```diff
@@ -85,7 +85,7 @@ as appropriate.
 By default, Sqitch will read F<sqitch.conf> in the current directory for
 settings. But it will also read F<~/.sqitch/sqitch.conf> for user-specific
 settings. Since MySQL's
-L<C<mysql> client|https://dev.mysql.com/doc/refman/5.6/en/mysql.html> is not
+L<C<mysql client|https://dev.mysql.com/doc/refman/5.7/en/mysql.html> is not
```

Comment:
The closing `>` after `mysql` is queried to complete `C<mysql>`.

Reply:
OK, fixed. Enclosed the whole "mysql client".